src/components/NotificationPanel/NotificationBox.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./NotificationBox.css";
+import PropTypes from "prop-types";
+
+const NotificationBox = ({ text, color, time, image }) => (
+  <div className="NotificationBox">
```

Comment:
I think the thing I told you last time with replacing `color` and `image` with `type` can be done here. Like this component should store a global variable that contains the mapping from `type` to `color` and `image`. Something like this:
```
const notificationTypes = {
  makePost: {
    type: "",
    color: "",
  },
  notificationType2: {...}
}
```
This way this component also only takes in type. This just makes sure interfaces between components are as clean as possible.

Reply:
couldn't think of how to do this so just added a componentDidMount and made this a bunch of if statements

Reply:
changed how i implemented it but still uses if statements :*(